toontown/estate/DistributedFurnitureManagerAI.py

```diff
@@ -587,3 +590,67 @@ def emptyHouse():
                 fm.items.remove(item)
             fm.saveToHouse()
     return "The house is empty"
+
+
+@magicWord(category=CATEGORY_PROGRAMMER, types=[int])
+def furniture(value):
+    """
+    ship any furniture item from your catalog
+    """
+
+    value = int(value)
+
+    target = spellbook.getTarget()
```

Comment:
The checks after this line aren't necessary. getTarget will either return the target, and if there is no target, it will return the invoker. getTarget() should also never return None.